docs/data_dictionary_hfs.md

```diff
@@ -0,0 +1,5 @@
+---
+id: 'data_dictionary_hfs'
+title: 'HFS'
+---
+
```

Comment:
Are you adding the diagram later?

Reply:
yes, it's the same component so using the Repairs page as the guinea pig